framework/PostProcessors/ComparisonStatisticsModule.py

```diff
@@ -179,10 +179,14 @@ def __processData(data, methodInfo):
   ret['counts'] = counts
   ret.update(mathUtils.calculateStats(sortedData))
   skewness = ret["skewness"]
-  delta = math.sqrt((math.pi / 2.0) * (abs(skewness) ** (2.0 / 3.0)) /
-                    (abs(skewness) ** (2.0 / 3.0) + ((4.0 - math.pi) / 2.0) ** (2.0 / 3.0)))
-  delta = math.copysign(delta, skewness)
-  alpha = delta / math.sqrt(1.0 - delta ** 2)
+  delta_func = lambda skewness: math.sqrt((math.pi / 2.0) * (abs(skewness) ** (2.0 / 3.0)) /
+                                (abs(skewness) ** (2.0 / 3.0) + ((4.0 - math.pi) / 2.0) ** (2.0 / 3.0)))
+  delta       = delta_func(skewness)
+  # see https://en.wikipedia.org/wiki/Skew_normal_distribution (Estimation)
+  delta_alpha = delta_func(min(0.9952717,skewness))
+  delta       = math.copysign(delta, skewness)
```

Comment:
do u need this line? you have already defined delta_alpha

Reply:
fixed